### DISCUSSION TITLE
Misalignment when multiple editor instance exist on same page

### DISCUSSION THREAD

**Discussion by siddhi1397** [GrapesJS/grapesjs#5575] [Q&A]
Whenever there are multiple instances of editor on the same page toolbar and body outline gets misaligned on scroll or other way.
I have added listenToEl also but not working for multiple editor. Also, provided each instance unique id.
@artf can you please help me with that?

**Comment by artf** [comment]
please provide a reproducible demo on jsfiddle (or similar) first

**Reply by siddhi1397** [reply]
PFA @artf 
[screencast-localhost_4200-2023.12.26-11_22_26.webm](https://github.com/GrapesJS/grapesjs/assets/121800257/bf357ff0-f576-4fee-a0eb-e6e6596bb6e4)

**Reply by artf** [reply]
video doesn't help here
